### ISSUE TITLE
Bug: Camera based barcode scanning is not working

### ISSUE THREAD

**Issue by TheHomekitWayFinder** [grocy/grocy#1502]
Camera based barcode scanning is not working properly. When clicking on the button to scan the barcode it opens a pop-up but never asks for access to the camera and only provides a drop down with options for barcodes. I am using the grocy addon in home-assistant so i dont know whether its a specific issue with the home assistant part or grocy itself. Both the grocy add-on and home assistant are up to date. Any help would be greatly appreciated.

Edit: Moved main issue description from title into the body and removed general bug report text.

**Comment by berrnd** [comment]
This is definitely works, please try to reproduce problems first on the pre-release demo like also mentioned on the issue template.

It has been mentioned on many other issues and on the [subreddit](https://www.reddit.com/r/grocy) that camera barcode scanning seems not be possible in when installed as an Home Assistant add-on - personally can't help for that, since I've never used HA and the better place for general help is the [subreddit](https://www.reddit.com/r/grocy) like also [mentioned in README](https://github.com/grocy/grocy#questions--help--bug-reporting--feature-requests) (or maybe also the HA Community for that).

**Comment by cat-inGlasses** [comment]
Hi, I have the same issue.
There are several errors, I've found in browser console:

1. After page loaded:
`[Error] SyntaxError: The string did not match the expected pattern.
	open (grocy.js:35)
	(anonymous function) (grocy.js:35)
	Global Code (grocy_dbchangedhandling.js:3)
[Error] SyntaxError: The string did not match the expected pattern.
	open (grocy.js:110)
	(anonymous function) (grocy.js:110)
	(anonymous function) (grocy.js:440)
	(anonymous function) (grocy.js:530)
	dispatch (jquery.min.js:2:43096)
	trigger (jquery.min.js:2:71519)
	(anonymous function) (jquery.min.js:2:72116)
	each (jquery.min.js:2:2981)
	Global Code (grocy_nightmode.js:64)
[Error] Failed to load resource: the server responded with a status of 404 (Not Found) (bootstrap.bundle.min.js.map, line 0)
[Error] Failed to load resource: the server responded with a status of 404 (Not Found) (toastr.js.map, line 0)
[Error] Failed to load resource: the server responded with a status of 404 (Not Found) (moment.min.js.map, line 0)
[Error] Failed to load resource: the server responded with a status of 404 (Not Found) (summernote-bs4.js.map, line 0)
[Error] Failed to load resource: the server responded with a status of 404 (Not Found) (sprintf.min.js.map, line 0)
[Error] Failed to load resource: the server responded with a status of 404 (Not Found) (bootstrap-select.min.js.map, line 0)`

2. After clicking "Photo" icon in bar code field:
[Error] Unhandled Promise Rejection: Error: enumerateDevices is not defined. This may mean that the user has declined camera access, or the browser does not support media APIs. If you are running in iOS, you must use Safari.
	(anonymous function) (barcodescanner.js:260)
	asyncFunctionResume
	(anonymous function)
	promiseReactionJobWithoutPromise


I've tried:
1. Several browsers: Google chrome, Firefox Developer Edition, Safari.
2. Launch Grocy from inside home assistant (by clicking on "Grocy" menu), and separately by getting to Grocy by ip and port (in my case 192.168.0.24:9192, port was changed in add-on settings.

In all cases I had the same problem. Also I've tried DEMO from Readme, and it worked as expected (there was opened window with camera view.

Also it doesn't work from mobile (iOS) and desktop (Mac) apps..

P.S. also SSL is set to false.
